src/librustc_metadata/native_libs.rs

```diff
@@ -163,7 +163,7 @@ impl<'a, 'tcx> Collector<'a, 'tcx> {
            !self.tcx.features().static_nobundle {
             feature_gate::emit_feature_err(&self.tcx.sess.parse_sess,
                                            "static_nobundle",
-                                           span.unwrap(),
+                                           span.unwrap_or_else(|| syntax_pos::DUMMY_SP),
```

Comment:
This should be `span.unwrap_or(syntax_pos::DUMMY_SP)`, as it's just a constant.

Reply:
Oh. I didn't even notice.